gulp-tasks/wfHelper.js

```diff
@@ -49,11 +52,11 @@ function promisedExec(cmd, cwd) {
       stdOut = stdOut.trim();
       stdErr = stdErr.trim();
       if (err) {
-        gutil.log(' ', cmdLog, chalk.red('FAILED'));
+        gutil.log('    (CMD):', chalk.red('FAILED'));
         reject(err);
         return;
       }
-      gutil.log(' ', cmdLog, chalk.green('OK'));
```

Comment:
Why remove this? If several commands are running asynchronously, you won't know which one failed.

Reply:
Added back in.